samples/scripts/vnet.fsx

```diff
@@ -2,19 +2,26 @@
 
 open Farmer
 open Farmer.Builders
+open Farmer.Network
+
+let serviceEndpointPolicy = Farmer.Arm.Network.serviceEndpointPolicies.resourceId "svc-endpt-policy"
+
 
 let privateNet = vnet {
     name "my-vnet"
     build_address_spaces [
         addressSpace {
             space "10.28.0.0/16"
+            build_subnet "services" 24
+            build_subnet "corporate-west" 18
+            build_subnet "corporate-east" 18
+            build_subnet "GatewaySubnet" 29
+            build_subnet_delegated "containers" 27 [ SubnetDelegationService.ContainerGroups ]
+            build_subnet_service_endpoints "can-use-servicebus" 28 [ EndpointServiceType.ServiceBus, [Location.EastUS] ]
+            build_subnet_service_endpoint_policies "can-use-storage" 28 [ EndpointServiceType.Storage, [Location.EastUS] ] [ serviceEndpointPolicy ]
```

Comment:
Consider combining `build_subnet_service_endpoints` and `build_subnet_service_endpoints_policies` e.g.

```fsharp
build_subnet_service_endpoints "can-use-servicebus" 28 (Endpoints.create([ EndpointServiceType.ServiceBus, [Location.EastUS] ], (* optional arg here *) )
```